model/Transmission/PerHost.cpp

```diff
@@ -63,12 +71,15 @@ PerHost::PerHost () :
 
 void PerHost::initialise (LocalRng& rng, double availabilityFactor) {
     relativeAvailabilityHet = availabilityFactor;
+    anophEntoAvailabilityRaw.resize(PerHostAnophParams::numSpecies());
     anophEntoAvailability.resize(PerHostAnophParams::numSpecies());
     anophProbMosqBiting.resize(PerHostAnophParams::numSpecies());
     anophProbMosqResting.resize(PerHostAnophParams::numSpecies());
+
     for(size_t i = 0; i < PerHostAnophParams::numSpecies(); ++i) {
         const PerHostAnophParams& base = PerHostAnophParams::get(i);
-        anophEntoAvailability[i] = base.entoAvailability->sample(rng) * availabilityFactor;
+        anophEntoAvailabilityRaw[i] = std::min(base.entoAvailability->sample(rng), 25.0);
```

Comment:
This is very arbitrary at the moment. This also breaks the HetVecDailyEIR test using the lognormal distribution.